teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super(20)
+    @specialization = specialization
+  end
```

Comment:
- [ ] [REQUIRED] Good job so far! It would be nice if you add `age, specialization, name = "Unknown", parent_permission = true`  to your `def` 🙏🏼 

```ruby
  def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
```